src/Resources/Locales/en.json

```diff
@@ -608,6 +608,64 @@
             "libraryName": "Panel",
             "description": "A layout panel"
         },
+        "dataHistory": {
+            "title": "Data history widget",
+            "libraryName": "Data history",
+            "description": "Show timeseries data in context of your 3D scene",
+            "notEnabledDescription": "Data history is not enabled for this ADT instance.",
+            "form": {
+                "connectionStringTitle": "Connection string",
+                "connectionStringPlaceholder": "Enter connection string (e.g. key1=value1;key2=value2;key3=value3)",
```

Comment:
this is a downside of having a single string stored for the connection string. The user now has to know the format and structure of the connection string rather than being able to drop in the database name and such and letting us create the right format.

Reply:
what do you think @darsney ? How should we store the connection string in the config file?

Reply:
decided to change the config schema in separate PR to keep those in separate fields.